source/views.txt

```diff
@@ -82,16 +80,16 @@ To open a view, either:
 
 - Click the desired view in the left-hand navigation.
 
-After you open a view, |compass-short| shows you that
-view's :doc:`Documents Tab </documents>`. |compass-short| provides the following
+After you open a view, |compass-short| shows you that view's :ref:`Documents Tab 
+<compass-documents>`. |compass-short| provides the following
 information and functionality for the view:
 
-- :doc:`/documents`
-- :ref:`compass-agg-builder` to aggregate based on your view
-- :doc:`/query-plan`
-- :doc:`/indexes`
-- :doc:`/schema`
-- :doc:`/validation`
+- :ref:`Document management <compass-documents>`
+- :ref:`Aggregation Pipeline Builder <compass-agg-builder>`
+- :ref:`Explain plans <explain-plans>`
```

Comment:
The current version of Compass (v1.39) do not have Explain tab anymore. We moved the entire functionality inside a button added to the query bar itself in v1.38.1. If this info still applies to this change (I guess it does) then we might be better without this list item.

Reply:
Noted and updated!